AnimationPlayground.playground/Contents.swift

```diff
@@ -0,0 +1,437 @@
+import UIKit
+import XCPlayground
+
+let liveViewWidth :CGFloat = 400.0
+let liveView = UIView(frame: CGRect(x: 0.0, y: 0.0, width: liveViewWidth, height: 2000.0))
+liveView.backgroundColor = UIColor.whiteColor()
+XCPlaygroundPage.currentPage.liveView = liveView
+
+
+
+// Express position/widths etc in relation to the view's width (i.e. "lineWidth = w * 0.02" instead of "lineWidth = 3") so all drawing scales proportionally with the animationView's size
+
+
+
+struct WMF_Welcome_Plus {
```

Comment:
we haven't explicitly decided on a swift coding style yet, but we should at least try to follow Apple & community conventions:

- No prefixes
- UpperCase classes, camelCase vars, fields, & methods